### PR TITLE
Show output of ctest runs

### PR DESCRIPTION
**Pull request by eile**
Needed to see what's going on with https://bbpcode.epfl.ch/ci/job/viz.ISC.gerrit/342/build_type=Debug,platform=Ubuntu-14.04/console

Might revert later if it's too annoying.

### CODE REVIEW

File: CommonCPPCTest.cmake [hunk 1]
```diff
@@ -121,9 +121,14 @@ foreach(FILE ${TEST_FILES})
   endif()
 endforeach()
 
+set(__CONSOLE)
+if(CMAKE_VERSION VERSION_GREATER 3.2.2)
```
Comment (tribal-tec) [inline CommonCPPCTest.cmake]: imo 3.2 is enough

**Comment by tribal-tec** [comment]
+1

File: CommonCPPCTest.cmake (hunk 1 above)
Comment (eile) [inline CommonCPPCTest.cmake]: Sure? I was it as new feature in 3.2.3, but maybe they have the same relnotes for all 3.2.x

Comment (rdumusc) [inline CommonCPPCTest.cmake]: Seems like 3.2 is enough: http://www.cmake.org/pipermail/cmake/2015-March/060082.html

**Comment by rdumusc** [comment]
+1

File: CommonCPPCTest.cmake (hunk 1 above)
Comment (eile) [inline CommonCPPCTest.cmake]: Ok, will change and then merge. Thanks.

**Pull request merged**